### ISSUE TITLE
Would like to add global_40_house_rules

### ISSUE THREAD

**Issue by beelee1** [triplea-maps/Project#92]

**Comment by DanVanAtta** [comment]
hi @beelee1  where is the map hosted?

**Comment by beelee1** [comment]
hi @DanVanAtta 
It is located here:
[https://github.com/beelee1/global_40_house_rules](url)

**Comment by beelee1** [comment]
well...hope I'm not being too impatient. Do i need to do something different ? I followed the guide at the triplea site here:

[https://docs.google.com/document/d/1FfF7N0srp9QG0_if5D-c1d1Aa1QTttdhxgm1GBh3pI4/edit?usp=sharing](url)

I guess I'll do the @DanVanAtta  and @ron-murhammer  since you guys are listed here.  I've spent a lot of time on this and would appreciate any information on how to proceed.

Thanks
